src/status_im2/contexts/wallet/home/view.cljs

```diff
@@ -57,6 +57,9 @@
 (defn view
   []
   (rf/dispatch [:wallet/get-wallet-token])
+  (rf/dispatch [:wallet/request-collectibles
+                {:offset       0
```

Comment:
what's offset?

Reply:
It is for retrieving collectibles by portions from backend. We start from zero and when receive the result - fetching more. In the event `:wallet/owned-collectibles-filtering-done` you can find:
```clojure
(when has-more
         [:dispatch
          [:wallet/request-collectibles
           {:offset next-offset}]])
```

Reply:
ok thanks for the explanation. imo "offset" is not descriptive of this. Perhaps "batch" or "portion" or something to that effect is clearer here? Np if you don't feel it needs changing just giving my 2c

Reply:
Yeah, I took this name from `status-go`, where arg is named `offset`. I think when you know the context the name seem okay. `batch` of `portion` for me sound more like the size of the batch that should be retrieved, whereas `offset` is a position from which retrieving should start. 
Wdyt about name `start-at`? Should sound logical when you read `[:wallet/request-collectibles {:start-at next-offset}]]`

Reply:
Yeah start-at or start-at-index or something like that is much clearer 👌